webservice/template/admin/assignment/single.tmpl

```diff
@@ -0,0 +1,63 @@
+{{define "title"}}Assignment: {{.Assignment.Name}}{{end}}
+{{define "head"}}{{end}}
+{{define "content"}}
+<h1 class="display-4">{{.Assignment.Name}}</h1>
+<hr>
+<div class="row">
+    <div class="col">
+        <table class="table">
+            <thead>
+            <tr>
+                <th>Key</th>
+                <th>Value</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td><strong>Name</strong></td>
+                <td>{{.Assignment.Name}}</td>
+            </tr>
+
+            <tr>
+                <td><strong>Created</strong></td>
+                <td>{{.Assignment.Created}}</td>
+            </tr>
+
+            <tr>
+                <td><strong>Publish</strong></td>
+                <td>{{.Assignment.Publish}}</td>
+            </tr>
+
+            <tr>
+                <td><strong>Deadline</strong></td>
+                <td>{{.Assignment.Deadline}}</td>
+            </tr>
+
+            <tr>
+                <td><strong>Course</strong></td>
+                <td>{{.Assignment.CourseID}}</td>
+            </tr>
+
+            <tr>
+                <td><strong>Submission Form</strong></td>
+                <td>{{.Assignment.SubmissionID}}</td>
+            </tr>
+
+            <tr>
+                <td><strong>Review From</strong></td>
+                <td>{{.Assignment.ReviewID}}</td>
```

Comment:
Maybe make this prettier? but is not a must

Reply:
Non-essential at this moment